.vscode/tasks.json

```diff
@@ -1329,6 +1338,23 @@
         ],
         "problemMatcher": []
       },
+      {
+        "label": "gen isar.protocol.extensions.anchoring",
+        "options": {
+          "cwd": "${workspaceFolder}/remoting/extras/isar.protocol.extensions"
+        },
+        "command": "flatc",
+        "args": [
+          "--cpp",
+          "--scoped-enums",
+          // "--filename-suffix", "",
+          "--cpp-str-flex-ctor",
+          "-o",
+          "isar.protocol/",
+          "anchoring.fbs"

```

Comment:
Multiple flatbuffers files can be generated with a single task. See https://github.com/Holo-Light/webrtc/commit/0ffe9dc36796172efc094327bcca3e3f48c9698a for how to do it.
Note: you do not need to pass the -I option, this is for includes which is no longer relevant

Reply:
Fixed: 5f7bfb74e87a4e08e0a3be3e6f7b0f7bb64388d5